easybuild/easyconfigs/m/MATES/MATES-0.1.5-20241121-foss-2023b.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'PythonBundle'
+
+name = 'MATES'
+version = '0.1.5-20241121'
+local_commit = '3846ad5'
+
+homepage = 'https://github.com/mcgilldinglab/MATES'
+description = "A Deep Learning-Based Model for Quantifying Transposable Elements in Single-Cell Sequencing Data."
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('Python-bundle-PyPI', '2023.10'),
+    ('SciPy-bundle', '2023.11'),
+    ('matplotlib', '3.8.2'),
+    ('anndata', '0.11.1'),
+    ('pybedtools', '0.10.0'),
+    ('PyTorch', '2.1.2'),
+    ('Pysam', '0.22.0'),
+    ('tqdm', '4.66.2'),
```

Comment:
@pavelToman the previous versions included `SAMtools` as a dep - is there a specific reason why you removed it?